### PR TITLE
Prebuilt image enhancements

### PR DESCRIPTION
**Pull request by FlyGoat**
Build all image variants, fix OAI image build, and provide document instructions.

Maybe it's worthy to cherry-pick those changes to all branches, so images will be built for all configurations.

### CODE REVIEW

**Comment by herlesupreeth** [comment]
Amazing contribution.. thanks a lot.

File: oai/enb_dockerfile [hunk 1]
```diff
@@ -27,6 +27,8 @@
 FROM ubuntu:focal
 
 ENV DEBIAN_FRONTEND=noninteractive
+ENV BUILD_UHD_FROM_SOURCE=True
+ENV UHD_VERSION=4.4.0.0
```
Comment (herlesupreeth) [inline oai/enb_dockerfile:31]: Is there a possibility to not hardcode UHD version??

File: oai/enb_dockerfile [hunk 2]
```diff
@@ -48,13 +50,14 @@ RUN git clone https://github.com/myriadrf/LimeSuite.git && \
...
 RUN git clone https://gitlab.eurecom.fr/oai/openairinterface5g.git && \
-    cd openairinterface5g/ && git checkout develop
+    cd openairinterface5g/ && git checkout v2.1.0
 
 # Set the working directory to openairinterface5g
 WORKDIR openairinterface5g
 
 RUN . ./oaienv && cd cmake_targets && \
-    ./build_oai -I -w USRP --eNB --verbose-compile
+    ./build_oai -I -w USRP --eNB --verbose-compile --disable-hardware-dependency
```
Comment (herlesupreeth) [inline oai/enb_dockerfile:59]: any reason why --disable-hardware-dependency was added?

Comment (FlyGoat) [reply oai/enb_dockerfile]: > any reason why --disable-hardware-dependency was added?

My bad, it was a leftover for my local debug, fixed.

File: oai/enb_dockerfile (hunk 1 above)
Comment (FlyGoat) [reply oai/enb_dockerfile:31]: Seems no, the build script requires this variable to be set.

4.4 was chosen by various official docker images, see:  https://gitlab.eurecom.fr/oai/openairinterface5g/-/blob/develop/docker/Dockerfile.base.ubuntu22?ref_type=heads#L34

**Comment by herlesupreeth** [comment]
@FlyGoat Hey, I will merge this once I am back from vacation

**Pull request merged**